src/viam/services/navigation/client.py

```diff
@@ -36,6 +39,11 @@ def __init__(self, name: str, channel: Channel):
         self.client = NavigationServiceStub(channel)
         super().__init__(name)
 
+    async def get_paths(self, *, timeout: Optional[float] = None) -> List[Path]:
```

Comment:
[q] Why do we return API types to users? I thought we would convert the API type to some native Python class and return that to the user?

Reply:
I want to request that follow-up API cleanup that doesn't have to do with Martha's readings change be handled in a follow-up ticket. This design does look wrong, though ) I think it should return a Python class but I don't want to make a decision given we're doing this on the fly.

Reply:
It's not bad to return a protobuf message. It's actually preferred IMO. This way we don't have to create a bunch of classes that mirror the proto, nor we do we have to keep those classes updated.

Reply:
@njooma that makes sense. Per recent conversation in the SDKs channel, it seems like it's just language-dependent whether the protos are "good enough" to not need native-mirrored types.